aptos-indexer-processors-sdk/sdk/src/common_steps/insertion_limit_step.rs

```diff
@@ -0,0 +1,298 @@
+use crate::{
+    traits::{AsyncRunType, AsyncStep, NamedStep, Processable},
+    types::transaction_context::TransactionContext,
+    utils::{
+        errors::ProcessorError,
+        step_metrics::{StepMetricLabels, INSERTION_LIMIT_STEP_REMAINING_BYTES},
+    },
+};
+use serde::{Deserialize, Serialize};
+use std::{
+    marker::PhantomData,
+    time::{Duration, Instant},
+};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct InsertionLimitConfig {
```

Comment:
Would be helpful to maybe above provide an example like:
`num_bytes=300, num_seconds=5` means that the processor can write up to 300 bytes per 5 min bucket

Reply:
can also consider adding `Rate` to the struct name like `InsertionLimitRateConfig`